dataflow/gemma-flex-template/metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Streaming Gemma-on-Dataflow",
+    "description": "A flex template configuring a streaming Gemma pipeline using pytorch and pub/sub",
+    "parameters": [
+      {
+        "name": "messages_subscription",
+        "label": "Input messages subscription",
+        "helpText": "The pub/sub subscription to pull inputs from"
+      },
+      {
+        "name": "responses_topic",
+        "label": "Output messages topic",
+        "helpText": "The pub/sub topic to push outputs to"
+      },
+      {
+        "name": "device",
+        "label": "Device",
+        "helpText": "The device to run inference on. Can be 'CPU' or 'GPU'."
+      }
```

Comment:
Per other comments, this file may need to be extended, but unclear.

Reply:
should be okay to leave as-is as explained above